rasa/shared/importers/importer.py

```diff
@@ -30,24 +30,15 @@ def get_domain(self) -> Domain:
         """
         raise NotImplementedError()
 
-    def get_stories(
-        self,
-        template_variables: Optional[Dict] = None,
-        use_e2e: bool = False,
-        exclusion_percentage: Optional[int] = None,
-    ) -> StoryGraph:
+    def get_stories(self, exclusion_percentage: Optional[int] = None,) -> StoryGraph:
```

Comment:
thanks for taking care of this 💯 As this is an interface which allows customizable implementations: can you please add a dedicated removal changelog for this? I also believe we show an example in the docs which potentially needs adapting